HighwayPatrol.cpp

```diff
@@ -28,7 +33,7 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     if( willArrest )
     {
         //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        std::cout << name << ": YOU IN THE [ " << v->name<< " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
```

Comment:
`v->name` is not the vehicle type.
- [ ] revise according to spec.

Reply:
need a bit of help here not sure how to print the actual object type.. car, motorcycle, semitruck etc..